region/brinstar/blue.json

```diff
@@ -571,9 +609,67 @@
             },
             {
               "id": 2,
-              "requires": [ "canUsePowerBombs" ],
+              "requires": [
+                {"or":[
+                  {"and":[
+                    "canUsePowerBombs",
+                    {"cost":[
+                      {"ammo": [
+                        {
+                          "type": "PowerBomb",
+                          "count": 1
+                        }
+                      ]}
+                    ]},
+                    {"obstacle": {
+                      "id": "A",
+                      "requires": [
+                        {"or":[
+                          "ScrewAttack",
+                          "Bombs",
+                          {"canShineCharge": {
+                            "usedTiles": 31,
+                            "shinesparkFrames": 43,
+                            "openEnd": 2
+                          }},
+                          {"cost":[
+                            {"ammo": [
+                              {
+                                "type": "PowerBomb",
+                                "count": 1
+                              }
+                            ]}
+                          ]}
+                        ]}
+                      ]
+                    }}
+                  ]},
+                  {"and":[
+                    "Morph",
+                    {"obstacle": {
+                      "id": "B",
+                      "requires": "never"
+                    }},
+                    {"or":[
+                      "ScrewAttack",
+                      "Bombs",
+                      {"and":[
+                        "PowerBomb",
+                        {"cost":[
+                          {"ammo": [
+                            {
+                              "type": "PowerBomb",
+                              "count": 1
```

Comment:
This req section should be within Obstacle "A" for the possibility to save 1 Power Bomb if starting from Boulder Bridge originally.

Reply:
Absolutely